infra/localnet/docker-compose.localnet.yml

```diff
@@ -0,0 +1,206 @@
+version: '3'
+
```

Comment:
This reminds me of how I tried to convince Denis before to make reusable compose files like this: https://github.com/consensus-shipyard/fendermint/pull/222#issuecomment-1717302364